lib/ntbb-session.lib.php

```diff
@@ -471,7 +471,12 @@ function getAssertion(string $userid, string $serverhostname, array $user = null
 		if (($user['userid'] === $userid) && !empty($user['loggedin'])) {
 			// already logged in
 			$usertype = '2';
-			if (in_array($userid, $psconfig['sysops'], true)) {
+			$ismain = $serverhostname === 'sim3.psim.us' || $serverhostname === 'smogtours.psim.us';
```

Comment:
This doesn't seem secure. Wouldn't this let you get sysop on Main by connecting through the IP address?

Reply:
Oh. Yes, if one is already a leader. Is there a good way around this?

Reply:
Pass a server ID here?

Reply:
We discussed in DMs and determined this actually wouldn't be an issue due to other guards against using IPs.